### PR TITLE
Added Utility Fns and Uninstallation option.

### PR DESCRIPTION
**Pull request by semi710**
added `check` fn to clean out the code.
and now install also handles uninstallation

### CODE REVIEW

File: devexcuse.sh [hunk 1]
```diff
@@ -1,21 +1,37 @@
 #!/usr/bin/env bash
 
+check() {
+	# Performs check if a command exists in the system
+	# sets the `$?` to 0 if the command exists, 1 otherwise
+	command -v "$1" &>/dev/null
+}
```
Comment (K4R7IK) [inline devexcuse.sh]: Could have integrated this in `check_dependencies` function

File: devexcuse.sh [hunk 2]
```diff
@@ -1,21 +1,37 @@
...
-  echo "jq is not installed."
-  exit 1
+	echo "Installing Excuse database..."
+	curl -fsSL https://raw.githubusercontent.com/K4R7IK/DevExcuse/master/excuse.json -o "$DATAFILE"
 fi
 
-get_random_excuse(){
- jq -r '.[] | select(.text_en) | .text_en' $DATAFILE | shuf -n 1
+get_random_excuse() {
+	jq -r '.[] | select(.text_en) | .text_en' "$DATAFILE" | shuf -n 1
```
Comment (K4R7IK) [inline devexcuse.sh:28]: Random space added for no reason.

Comment (semi710) [reply devexcuse.sh:28]: Done by shfmt (standard shell formatting)

**Review by K4R7IK — CHANGES_REQUESTED** [review]
Integrate the check function in check_dependecies.

**Review by K4R7IK — APPROVED** [review]

**Pull request merged**